ensure_tiller_installed.go

```diff
@@ -140,7 +140,7 @@ func (c *Client) EnsureTillerInstalledWithValues(ctx context.Context, values []s
 			Rules: []rbacv1.PolicyRule{
 				{
 					APIGroups: []string{
-						"extensions",
+						"policy",
```

Comment:
What if TC/CPs are still based on 1.14.x version k8s? is that still okay?

Reply:
Yes, because policy/v1beta1 is available since 1.10 (see https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/)